--- a/bat.spec
+++ b/bat.spec
@@ -495,8 +495,8 @@
 %install
 %cargo_install
 # install man page
-install -Dpm0644 target/release/build/%{crate}-*/out/assets/manual/bat.1 \
-    -t %{buildroot}%{_mandir}/man1/
+find target/release/build -name bat.1 -print -quit | xargs -I {} install -Dpm0644 {} \
+    %{buildroot}%{_mandir}/man1/bat.1
 # install shell completions
 install -Dpm0644 target/release/build/%{crate}-*/out/assets/completions/bat.bash \
     %{buildroot}/%{bash_completions_dir}/bat.bash
