--- libbpf/src/bpf_helpers.h.orig	2026-07-01 17:03:45.197793268 +0200
+++ libbpf/src/bpf_helpers.h	2026-07-01 17:03:45.199016874 +0200
@@ -315,7 +315,7 @@
 			  ___param, sizeof(___param));		\
 })
 
-extern int bpf_stream_vprintk(int stream_id, const char *fmt__str, const void *args,
+extern int bpf_stream_vprintk(int stream_id, const char *fmt__str, const void *args, u32 len__sz,
 			      __u32 len__sz, void *aux__prog) __weak __ksym;
 
 #define bpf_stream_printk(stream_id, fmt, args...)				\
